sputnikdao2/src/proposals.rs

```diff
@@ -41,6 +41,17 @@ pub struct ActionCall {
     gas: U64,
 }
 
+/// Function call arguments.
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize)]
+#[cfg_attr(not(target_arch = "wasm32"), derive(Clone, Debug))]
```

Comment:
don't think we need `Debug` here, but doesn't seem to affect the contract size

Reply:
Tried to mimic the `derive`s from `ProposalKind`, since `PolicyParameters` is used inside `ProposalKind`.

`ActionCall` (which is also used inside `ProposalKind`) also has the same `derive`s.